### ISSUE TITLE
"Text has enhanced contrast" 09o5cg failed example 2 fails "Text has minimum contrast"

### ISSUE THREAD

**Issue by dd8** [act-rules/act-rules.github.io#1666]
Failed example 2 for "Text has enhanced contrast" (WCAG 1.4.6) is a failure of "Text has minimum contrast" (WCAG 1.4.3 + WCAG 1.4.6) because some of the characters have contrast ratio 1.6

No other examples for "Text has enhanced contrast" fail "Text has minimum contrast"

>This light gray text has a contrast ratio between 1.6:1 and 5.3:1 on the white to blue gradient background.

```
<p style="color: #CCC; background: linear-gradient(to right, #FFF, #00F); width: 300px">
	Some text in English
</p>
```
https://act-rules.github.io/rules/09o5cg#failed-example-2

This is an issue for tools that reduce noise by not reporting a 1.4.6 enhanced contrast failure when a 1.4.3/1.4.6 minimum contrast failure has already been reported.

**Comment by WilcoFiers** [comment]
OK, so just so I'm clear. We need to increase the contrast of failed example 2 so that it only fails 1.4.6, but not 1.4.3.

**Comment by dd8** [comment]
> OK, so just so I'm clear. We need to increase the contrast of failed example 2 so that it only fails 1.4.6, but not 1.4.3.

Yes, exactly so.
